tchannel/__init__.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+from .tchannel import TChannel  # noqa
+from .thrift import from_thrift_module  # noqa
```

Comment:
my thinking is that they would do something like:

```
donut_service = from_thrift_module(...)
```

That actually reads out well enough IMO